README.md

```diff
@@ -78,29 +86,29 @@ This can be done by using any of the following alternative methods (documented i
 - <a id="user-auth-client-envar"></a>Set the authentication credentials in environment variables, by using either of the following methods:
 
   - <a id="user-auth-client-envar-access-key"></a>Set the `V3IO_ACCESS_KEY` environment variable to a valid platform access key with the required data-access permissions.
+
+    > **Note:** The platform's Jupyter Notebook service automatically defines the `V3IO_ACCESS_KEY` environment variable and initializes it to a valid access key for the running user of the service. 
   - <a id="user-auth-client-envar-user-pwd"></a>Set the `V3IO_USERNAME` and `V3IO_PASSWORD` environment variables to the username and password of a platform user with the required data-access permissions.
-  <br/>
 
   > **Note:**
-  > - When the client constructor is called with [authentication parameters](#user-auth-client-const-params), the authentication-credentials environment variables (if defined) are ignored.
+  > - When the client constructor is called with authentication parameters ([option #1](#user-auth-client-const-params)), the authentication-credentials environment variables (if defined) are ignored.
   > - When `V3IO_ACCESS_KEY` is defined, `V3IO_USERNAME` and `V3IO_PASSWORD` are ignored.
-  > - The platform's Jupyter Notebook service automatically defines the `V3IO_ACCESS_KEY` environment variable and initializes it to a valid access key for the running user of the service. 
 
 <a id="client-constructor"></a>
-### Client Constructor
+### `Client` Constructor
 
 All Frames operations are executed via an object of the `Client` class.
 
 - [Syntax](#client-constructor-syntax)
 - [Parameters and Data Members](#client-constructor-parameters)
 - [Return Value](#client-constructor-return-value)
-- [Example](#client-constructor-example)
+- [Examples](#client-constructor-examples)
 
 <a id="client-constructor-syntax"></a>
 #### Syntax
 
 ```python
-Client(address='', container='', user='', password='', token='')
+Client(address=""[, data_url=""], container=""[, user="", password="", token=""])
```

Comment:
move the `data_url` to the end with all the optional parameters

Reply:
REJECT — as we have just discussed (before I saw your comment), in Python there's a significance to the order of the parameters in the syntax, because the first parameters can be passed in the function call without specifying the parameter name. I'll add to my proposed API changes to change the order of the `Client` constructor params so that all mandatory params appear first.